R/menu.R

```diff
@@ -54,10 +55,10 @@ render_menu_link <- function(location, title, active_location = "", icon = NA) {
 #' @export
 #'
 #' @examples
-#' menu_content <- data.frame(
-#'  name = paste("Menu", 1:3),
-#'  link = paste("page", 1:3),
-#'  icon = c("home", "car", 'tree')
+#' menu_content <- list(
```

Comment:
An example is missing packages loading

 library(shiny.semantic)
 library(shiny)